subxt/src/config/mod.rs

```diff
@@ -65,6 +56,40 @@ pub trait Config: Sized + Send + Sync + 'static {
 /// given some [`Config`], this return the other params needed for its `ExtrinsicParams`.
 pub type OtherParamsFor<T> = <<T as Config>::ExtrinsicParams as ExtrinsicParams<T>>::OtherParams;
 
+/// Block hashes must conform to a bunch of things to be used in Subxt.
+pub trait BlockHash:
+    Debug
+    + Copy
+    + Send
+    + Sync
+    + Decode
+    + AsRef<[u8]>
+    + Serialize
+    + DeserializeOwned
+    + Encode
+    + PartialEq
+    + std::cmp::Eq
+    + std::cmp::PartialEq
```

Comment:
I think this trait bound might have PartialEq twice?

Reply:
Oh yeah, so it does! Good spot!